### ISSUE TITLE
Webhook Integration

### ISSUE THREAD

**Issue by ghost** [Zingle/rest-api-documentation#11]
Hi @njovin , 

Would you please take some time for me to help my another problem?

Currently, I am trying to integrate the webhook in order to get new messages automatically via API.
But it looks like there is only one support in the current webhook set up. That is, I can find only "Contact Opt-In Status Change" on the Webhooks page. 

Maybe it's not possible to get a notification automatically via Webhook? Or should I pull all messages using current API periodically? 

Thanks again for your help.

**Comment by njovin** [comment]
@bloolight That's correct, we only support webhooks for the contact opt-in status today.  There's not a way to get notified of new messages.

**Comment by ghost** [comment]
@njovin  I see, thanks for the confirmation. 

Much appreciate your help.
